src/ProcessMaker/Nayra/Bpmn/ProcessTrait.php

```diff
@@ -323,4 +324,29 @@ public function call(DataStoreInterface $dataStore = null)
         });
         return $instance;
     }
-}
\ No newline at end of file
+
+    /**
+     * Set the lane sets of the process
+     *
+     * @param CollectionInterface $laneSets
+     *
+     * @return $this
+     */
+    public function getLaneSets()
```

Comment:
Do this returns $this? it seems a collection

Reply:
Fixed. It returns a Collection of LaneSetInterface